src/main/java/com/bina/varsim/fastqLiftover/GenomeLocation.java

```diff
@@ -3,13 +3,20 @@
 public class GenomeLocation implements Comparable<GenomeLocation> {
     public String chromosome;
     public int location = 0;
+    public int read_location1 = -1; // if >=0, this is the 1-base index corresponding to a location in a read
```

Comment:
Haha, up to you. Feel free to refactor.



--John Mu


Sent from Mailbox

On Fri, Feb 27, 2015 at 5:12 PM, Bayo Lau <notifications@github.com>
wrote:

>> @@ -3,13 +3,20 @@
>>  public class GenomeLocation implements Comparable<GenomeLocation> {
>>      public String chromosome;
>>      public int location = 0;
>> +    public int read_location1 = -1; // if >=0, this is the 1-base index corresponding to a location in a read
> where should I place that class? seems pretty generic?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/bioinform/varsim/pull/24/files#r25551513